packages/os/bootstrap-containers@.service

```
@@ -0,0 +1,23 @@
+[Unit]
+Description=bootstrap container %i
+Before=configured.target
+After=host-containerd.service
```

Should also have a dependency:
```suggestion
After=host-containerd.service
Wants=host-containerd.service
```

---

Done